src/distilabel/steps/tasks/benchmarks/arena_hard.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from typing import Any, Dict, List, Optional, Union
+
+from distilabel.steps import GlobalStep, StepInput
+from distilabel.steps.tasks.base import Task
+from distilabel.steps.tasks.typing import ChatType
+from distilabel.steps.typing import StepOutput
+
+
+class ArenaHard(Task):
+    """This `Task` is based on the "From Live Data to High-Quality Benchmarks: The
+    Arena-Hard Pipeline" paper, which presents Arena Hard which is a benchmark for
+    instruction-tuned LLMs that contains 500 challenging user queries. GPT-4 is used
+    as the judge to compare the model responses against a baseline model, which defaults
+    to `gpt-4-0314`.
+
+    Note:
+        Arena-Hard-Auto has the highest correlation and separability to Chatbot Arena
+        among popular open-ended LLM benchmarks.
```

Comment:
```suggestion
        Arena-Hard-Auto has the highest correlation and separability of Chatbot Arena
        among popular open-ended LLM benchmarks.
```

Reply:
I believe in this case the correct form is `to` (also this text was a copy-paste with minimal edits from their official post), so not 100% sure either